portal/src/texts/profile/tone.md

```diff
@@ -1,51 +1,55 @@
 
-Eksempler i kommunikasjonen:
-I alle kanaler skal vi bruke mindre tekst og mer illustrasjoner.
-Vi stiller spørsmål og snakker så folk forstår.
-Gjør vi en feil, innrømmer vi det.
+#### Eksempler i kommunikasjonen
+
+- I alle kanaler skal vi bruke mindre tekst og mer illustrasjoner.
+- Vi stiller spørsmål og snakker så folk forstår.
+- Gjør vi en feil, innrømmer vi det.
 
 ### Varm
 
-Vi er medmennesker og tilfører både trygghet og varme i språket vårt
-Det å håndtere følelsesmessige oppturer og nedturer er en del av hverdagen vår. Vi møter mennesker som opplever uhell, trøbbel og som er i sorg. Vi er her for å hjelpe og forebygge.
+**Vi er medmennesker og tilfører både trygghet og varme i språket vårt**
+Det å håndtere følelsesmessige oppturer og nedturer er en del av hverdagen vår. Vi møter mennesker som opplever uhell, trøbbel og sorg. Vi er her for å hjelpe og forebygge.
 
-Vi er medmennesker og snakker med mennesket fremfor «kunden». Vi ser på alle som likeverdige og snakker med en varme i stemmen. Vi formidler en trygghetsfølelse gjennom å snakke rolig, slik at vårt ønske om å gjøre en grundig vurdering skinner igjennom når vi uttrykker oss.
+Vi er medmennesker og snakker med mennesket fremfor «kunden». Vi ser på alle som likeverdige og snakker med en varme i stemmen. Vi formidler en trygghetsfølelse gjennom å snakke rolig, slik at vårt ønske om å gjøre en grundig vurdering skinner gjennom når vi uttrykker oss.
 
-Eksempler i kommunikasjonen:
-Vi har ikke hastverk når vi snakker med en kunde, men er tydelig og rolig.
-Vi snakker «med» – og ikke «til».
-Vi tar kunden på alvor, sørger for at de ikke føler seg dumme.
-Vi skryter ikke av oss selv, men lar kundene fortelle om sine opplevelser.
+#### Eksempler i kommunikasjonen
+
+- Vi har ikke hastverk når vi snakker med en kunde, men er tydelig og rolig.
+- Vi snakker «med» – og ikke «til».
+- Vi tar kunden på alvorog sørger for at de ikke føler seg dumme.
+- Vi skryter ikke av oss selv, men lar kundene fortelle om sine opplevelser.
 
 ### Presis
 
-Vi skal bli verdensmestere i klarspråk
+**Vi skal bli verdensmestere i klarspråk**
 Vi snakker norsk, og bruker riktig grammatikk og språkregler.
 
 Vi er presise i alt vi sier, uansett om det gjelder hva forsikringen dekker, hvorfor vi samler inn data om kundene våre, eller hvordan vi gir kundene oversikt og full kontroll over forsikringene sine.
 
-Kundene skal oppleve at de får informasjon tilpasset akkurat seg, som er relevant, kommer til riktig tid og i riktig kanal.
+Kundene skal oppleve at de får informasjon tilpasset akkurat seg. Den skal være relevant, komme til riktig tid og i riktig kanal.
+
+#### Eksempler i kommunikasjonen
 
-Eksempler i kommunikasjonen:
-Vi bruker «Ti tips for å skrive bedre».
-Vi fjerner stammespråk.
-Vi hjelper hverandre med å lese korrektur på alle tekster.
+- Vi bruker «Ti tips for å skrive bedre».
+- Vi fjerner stammespråk.
+- Vi hjelper hverandre med å lese korrektur på alle tekster.
```

Comment:
My little pet peeve I guess, but I'd really like us to take a clear stance on horrible three letter abreviations.

My suggestion:
`Forkortelser er ikke lov, med mindre forkortelsen er mer forklarende enn ordet som blir forkortet`
Eks:
`SQL is okay, as its a more establish term than what is actually stand for`
`FDS is not okay, Fremtind Design System is easy to understand. FDS is nonsense`

Reply:
I agree that we should have examples like this and also a link to what plain language is. However, I am holding it off a bit, since we are still waiting for a style guide from Market/komm/Charlotte. I am not sure if it is on her agenda, but I will bring it up in the next "fagmiljø-møte", next Friday. I can make one, but have not been assigned that task yet.